po/fr.po

```diff
@@ -27376,129 +27426,134 @@ msgid "Force-push branch"
 msgstr "Pousser autoritairement la branche"
 
 #: /tmp/fish/implicit/share/completions/git.fish:226
+#, fuzzy
+msgid "Delete remote branch"
+msgstr "Supprimer la branche"
```

Comment:
la branche *distante*

the *remote* branch

Reply:
Indeed.